iothub/service/src/Messaging/ErrorContext.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
```

Comment:
Missing the copyright header that we put at the top of every source code file. I'd setup your Visual Studio instance to do this for you: https://docs.microsoft.com/en-us/visualstudio/ide/reference/add-file-header?view=vs-2022

Reply:
@timtay-microsoft  added header to all missing files.